packages/client/src/base/args.ts

```diff
@@ -13,4 +13,14 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
+import { SModelElement } from "sprotty";
+
 export type Args = { [key: string]: string | number | boolean };
+
+export interface SArgumentable {
+    args: Args;
+}
+
+export function hasArguments(element: SModelElement | undefined): element is SModelElement & SArgumentable {
```

Comment:
I would opt for the concise notation of optional arguments here:
```suggestion
export function hasArguments(element?: SModelElement): element is SModelElement & SArgumentable {
```